tests/test_definitions/multicolumn_map_expectations/expect_compound_columns_to_be_unique.json

```diff
@@ -0,0 +1,197 @@
+          },
+          {
+            "a": 1.0,
+            "b": 2.0
+          },
+          {
+            "a": 1.0,
+            "b": 1.0
+          },
+          {
+            "a": 2.0,
+            "b": 2.0
+          },
+          {
+            "a": 2.0,
+            "b": 1.0
+          },
+          {
+            "a": 2.0,
+            "b": 2.0
+          },
+          {
+            "a": 2.0,
+            "b": 1.0
+          }
+        ],
+        "unexpected_index_list": [0,1,2,3,4,5,6,7,8],
+        "success": false
+      },
+    "suppress_test_for": ["sqlalchemy", "sqlite", "postgresql", "mysql", "mssql", "spark"]
+    },{
+      "title": "Unexpected Values - exact match out",
+      "exact_match_out" : true,
+      "in": {
+        "column_list": ["a", "b"]
+      },
+      "out": {
+        "success": false,
+        "result": {
+          "element_count": 10,
+          "missing_count": 1,
+          "missing_percent": 10.0,
+          "unexpected_count": 9,
+          "unexpected_percent": 90.0,
+          "unexpected_percent_nonmissing": 100.0,
+          "partial_unexpected_list": [
+            {
+              "a": 1.0,
+              "b": 1.0
+            },
+            {
+              "a": 1.0,
+              "b": 2.0
+            },
+            {
+              "a": 1.0,
+              "b": 1.0
+            },
+            {
+              "a": 1.0,
+              "b": 2.0
+            },
+            {
+              "a": 1.0,
+              "b": 1.0
+            },
+            {
+              "a": 2.0,
+              "b": 2.0
+            },
+            {
+              "a": 2.0,
+              "b": 1.0
+            },
+            {
+              "a": 2.0,
+              "b": 2.0
+            },
+            {
+              "a": 2.0,
+              "b": 1.0
+            }
+          ],
+          "partial_unexpected_index_list": [0,1,2,3,4,5,6,7,8],
+          "partial_unexpected_counts": [
+            "partial_exception_counts requires a hashable type"
+          ],
+          "unexpected_list": [
+            {
+              "a": 1.0,
+              "b": 1.0
+            },
+            {
+              "a": 1.0,
+              "b": 2.0
+            },
+            {
+              "a": 1.0,
+              "b": 1.0
+            },
+            {
+              "a": 1.0,
+              "b": 2.0
+            },
+            {
+              "a": 1.0,
+              "b": 1.0
+            },
+            {
+              "a": 2.0,
+              "b": 2.0
+            },
+            {
+              "a": 2.0,
+              "b": 1.0
+            },
+            {
+              "a": 2.0,
+              "b": 2.0
+            },
+            {
+              "a": 2.0,
+              "b": 1.0
+            }
+          ],
+          "unexpected_index_list": [0,1,2,3,4,5,6,7,8]
+        }
+      },
+    "suppress_test_for": ["sqlalchemy", "sqlite", "postgresql", "mysql", "mssql", "spark"]
+    }]
+  }]
+}
```

Comment:
Curious: How was this JSON test file generated?  Automatically or manually?  Also, how come we are excluding databases.  Finally, if we are excluding databases, maybe just suppressing "sqlalchemy" would be sufficient?